Sprint-3/implement/get-angle-type.js

```diff
@@ -25,3 +25,27 @@
 // Identify Reflex Angles:
 // When the angle is greater than 180 degrees and less than 360 degrees,
 // Then the function should return "Reflex angle"
+
+
+function getAngleType(angle){
+    if(angle === 90){
+      return "Right angle";
+    }
+    
+    if(angle  < 90){
+      return "Acute angle";
+    }
+    if (angle < 180){
+      return "Obtuse angle";
+    }
+    
+    if(angle === 180){
+      return "Straight angle"
+    }
+  
+    if(angle < 360){
+      return "Reflex angle" 
+    }
+  }
```

Comment:
It will give us an angle number equal to or greater than 360. 
 
 Honestly, I was confused here because I just couldn't figure a way to code this statement " angle is greater than 180 degrees and less than 360 degrees" 
 
 so I thought I would just focus on the last part that says less than 360 degrees but I see that was only half of the answer please check I fixed my if statement.